File: src/main/java/me/ryzeon/transcripts/utils/format/impl/AudioFormat.java
package me.ryzeon.transcripts.utils.format.impl;

import me.ryzeon.transcripts.utils.format.IFormatHelper;

import java.util.Arrays;
import java.util.List;

public class AudioFormat implements IFormatHelper {

    final List<String> formats = Arrays.asList("mp3", "wav", "ogg", "flac");

    @Override
    public List<String> formats() {
        return formats;
    }
}

File: transcripts-core/src/main/kotlin/net/dungeonhub/transcripts/utils/format/impl/AudioFormat.kt
package net.dungeonhub.transcripts.utils.format.impl

import net.dungeonhub.transcripts.utils.format.IFormatHelper

class AudioFormat : IFormatHelper {
    val formats: MutableList<String> = mutableListOf("mp3", "wav", "ogg", "flac")

    override fun formats(): MutableList<String> {
        return formats
    }
}
